src/pages/cart-page/cart-list-product-item/cart-list-product-item.tsx

```diff
@@ -0,0 +1,109 @@
+import { Box, IconButton, useMediaQuery } from "@mui/material";
+import { FC, useState } from "react";
+import DrawerCartBoxItemControl from "../cart-control-product/cart-control-product";
+import { CartProductModel } from "../../../api/api-cart/api-get-cart";
+
+import CloseOutlinedIcon from "@mui/icons-material/CloseOutlined";
+
+interface CartListProductItemPropsModel {
+  productItem: CartProductModel;
+  cartContext: CartContextInterface;
+}
+import { CartContextInterface } from "../../../context/cart-context";
+import {
+  Item,
+  ItemImg,
+  ContentBox,
+  InfoBox,
+  HeadersTypography,
+  ChildrenHeadersTypography,
+  PriceTypography,
+} from "./cart-list-product-item.styled";
+import { breakpointViewSize } from "../../../state/state";
+
+const CartListProductItem: FC<CartListProductItemPropsModel> = ({
+  productItem,
+  cartContext,
+}) => {
+  const [cartItemQuantity, setCartItemQuantity] = useState<number>(
+    productItem.quantity
+  );
+  return (
+    <Item    
+      sx={{
+        width: useMediaQuery(breakpointViewSize)
```

Comment:
Is this work without throwing an error? React hook should not be used in such way. They should be always declared on the top of file.